profile.md

```diff
@@ -189,13 +189,27 @@ deployment:
   "space_guid": "aaaa1234-da91-4f12-8ffa-b51d0336aaaa"
   ```
 
+- `name`
```

Comment:
Since 'name' isn't platform specific (which is what I sort of view 'context' for), I'm wondering why we're not putting this into the main body of the request as an optional field?

Reply:
I kind of think `name` _is_ platform specific. There is no requirement for platforms to _name_ service instances (and I think some CLI tools exist that behave as platforms and don't have names).

Reply:
I probably didn't use the right phrase, I view things like org/space/namespace as environment-specific. While things like instanceID and name (while platform specific) aren't really about the environment as much as they're about the thing being operated on. I view context as "extra info to help provide, well, context about where the resource lives", not so much about the resource itself. The spec say:
- - -
Platform specific contextual information under which the Service Instance is to be provisioned.
- - -
Which I think this is consistent. If we put info about the resource itself I wonder where we draw the line then.... why not put service, plan and parameters under there too?